app/utils.py

```diff
@@ -382,9 +382,9 @@ def ensure_local_services_are_running() -> int:
                 log("Unable to connect to mysql server.", Ansi.LRED)
                 return 1
 
-    if not os.path.exists("/var/run/redis/redis-server.pid"):
-        log("Unable to connect to redis server.", Ansi.LRED)
-        return 1
+    # if not os.path.exists("/var/run/redis/redis-server.pid"):
+    #     log("Unable to connect to redis server.", Ansi.LRED)
+    #     return 1
```

Comment:
this stuff can all be removed once we have the healthchecks in the start.sh scriptr

Reply:
+1 on that